src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -92,7 +92,15 @@ export class GlobalMap extends React.Component<Props, State> {
   emitFilteredBucketResults() {
     // TODO: map region filtering can live here.
     const filter = this.filters[this.state.activeIndex]
-    EventEmitter.dispatch("map:change", { filter, buckets: this.state.bucketResults, relay: this.props.relay })
+    const {
+      city: { name: cityName },
+    } = this.props.viewer
+    EventEmitter.dispatch("map:change", {
```

Comment:
This may need some refactoring once we figure out the details in our data fetching strategy but let's go with it for now